### PR TITLE
IT PR

### PR DESCRIPTION
**Pull request by albertovlcbo**
IT - Translation of remaining .dtd files in “Mail” folder.

### CODE REVIEW

File: it/mail/chrome/messenger/phishing-afterload-warning-message.dtd [hunk 1]
```diff
@@ -1,28 +1,28 @@
-<!ENTITY safeb.palm.warning.title "Suspected Web Forgery">
+<!ENTITY safeb.palm.warning.title "Sospetta falsificazione web">
 
-<!ENTITY safeb.palm.message.p1 "This page has been reported as a web forgery designed to trick users into sharing personal or financial information. Entering any personal information on this page may result in identity theft or other fraud. &#160;">
-<!ENTITY safeb.palm.message.p1.linkText "Read more &#187;">
-<!ENTITY safeb.palm.p1.linkStatusText "Read more &#133;">
+<!ENTITY safeb.palm.message.p1 "Questa pagina è stata segnalata come falsificazione web realizzata per ingannare gli utenti e spingerli a condividere informazioni personali o finanziarie. Inserire qualsiasi informazione personale su questa pagina potrebbe casusare un furto d'identità o altra frode. &#160;">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/phishing-afterload-warning-message.dtd]: CORRECTIONS NEEDED. 

There is a typo. Please replace "casusare" with "causare".

File: it/mail/chrome/messenger/phishing-afterload-warning-message.dtd [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+<!ENTITY safeb.palm.message.p1.linkText "Continua a leggere &#187;">
+<!ENTITY safeb.palm.p1.linkStatusText "Continua a leggere &#133;">
 
-<!ENTITY safeb.palm.message.p2.start "These types of web forgeries are used in scams known as phishing attacks, in which fraudulent web pages and emails are used to imitate sources you may trust. You can find out more about ">
-<!ENTITY safeb.palm.message.p2.linkText "how &brandShortName; protects you">
-<!ENTITY safeb.palm.message.p2.end " from phishing attacks.">
+<!ENTITY safeb.palm.message.p2.start "Questi tipi di falsificazioni web sono utilizzate in truffe note come attacchi di phishing, in cui pagine web ed e-mail fraudulente sono utilizzate per imitare le risorse di cui ti potresti fidare. Puoi scoprire di più su ">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/phishing-afterload-warning-message.dtd]: CORRECTIONS NEEDED. 

There is a typo. Please replace "fraudulente" with "fraudolente".

File: it/mail/chrome/messenger/safeMode.dtd [hunk 3]
```diff
@@ -43,26 +43,26 @@
 <!ENTITY safeModeOptionsWindow.width       "width: 49em;">
 <!ENTITY safeModeOptionsDialog.title       "&brandShortName; Trouble Shooting">
 
-<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting 
+<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting
 with one or more of these trouble shooting options selected:">
 
 
-<!ENTITY disableAddons.label              "Permamently Disable all Add-ons">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/safeMode.dtd]: Please check the English source text and replace "Permamently" with "Permanently".

File: it/mail/chrome/messenger/subscribe.dtd [hunk 4]
```diff
@@ -35,10 +35,10 @@
 
  ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY subscribeDialog.title        "Sottoscrizioni">
-<!ENTITY subscribeButton.label        "Sottoscrivi">
+<!ENTITY subscribeDialog.title        "Iscriviti">
```
Comment (fenderman79) [inline it/mail/chrome/messenger/subscribe.dtd]: CORRECTIONS NEEDED. 

I would change "Iscriviti" with "Iscrizioni", to keep the same difference there was between the original sentences 38 and 39 (Sottoscrizioni-Sottoscrivi --> Iscrizioni-Iscriviti).

File: it/mail/chrome/messenger-newsblog/feed-subscriptions.dtd [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <!-- Subscription Dialog -->
 <!ENTITY newsBlogSubscriptions.label "RSS Subscriptions">
-<!ENTITY subscriptionDesc.label "Nota: rimuovere o cambiare la cartella per un feed non influenzerà gli articoli già scaricati.">
+<!ENTITY subscriptionDesc.label "Nota: rimuovere o cambiare la cartella di un feed non influenzerà sugli articoli già scaricati.">
```
Comment (fenderman79) [inline it/mail/chrome/messenger-newsblog/feed-subscriptions.dtd]: CORRECTIONS NEEDED. 

Please replace "influenzerà" with "avrà influenza".

File: it/mail/chrome/messenger/safeMode.dtd (hunk 3 above)
Comment (shermandickman) [inline it/mail/chrome/messenger/safeMode.dtd]: @albertovlcbo We need line 46 translated, suggestion is:
Se stai avendo difficoltà con Postbox, prova a riavviare con una o più delle opzioni per la risoluzione dei problemi selezionate:

File: it/mail/chrome/messenger/virtualFolderListDialog.dtd [hunk 6]
```diff
@@ -1,4 +1,2 @@
-<!ENTITY virtualFolderListTitle.title        "Selezionare la/e cartella/e">
-<!ENTITY virtualFolderDesc.label       "Selezionare le cartelle in cui cercare:">
-
-
```
Comment (shermandickman) [inline it/mail/chrome/messenger/virtualFolderListDialog.dtd]: @albertovlcbo Let's be careful about making changes that are not necessary, as it creates more work for people to review.

**Pull request merged**